custom_components/csnet_home/climate.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, hass, entry, sensor_data, common_data):
         self._attr_min_temp = HEATING_MIN_TEMPERATURE
         self._attr_max_temp = HEATING_MAX_TEMPERATURE
         self._attr_temperature_unit = UnitOfTemperature.CELSIUS
-        self._attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
+        self._attr_hvac_modes = [HVACMode.AUTO, HVACMode.OFF, HVACMode.HEAT, HVACMode.COOL]
```

Comment:
Is the `COOL` mode for the Air2Air PAC version?

Reply:
no, I have a water2air heatpump for cooling
That's why I also need cooling

Reply:
This part I think should be aligned with the model... actually I'm having things that are not supported by my heat pump. I will try to figure out if there is way.
<img width="419" alt="image" src="https://github.com/user-attachments/assets/f72412cb-1ccf-4153-9129-85b2535732b9" />

Reply:
@CrazYoshi can you check in this API call (https://www.csnetmanager.com/data/elements), what are the values for these parameters on your side?
```
"hasCooling": false,
"hasAuto": false,
"hasBoost": false,
"c1Demand": false,
"c2Demand": true,
"doingBoost": false,
"timerRunning": false,
"fixAvailable": false
```
If needed I can share the postman...

Reply:
This are my values for heating/cooling circuit:
 ```
"hasCooling": false,
"hasAuto": false,
"hasBoost": false,
"c1Demand": false,
"c2Demand": false,
"doingBoost": false,
"timerRunning": false,
"fixAvailable": false
```
for hwc instead:
```
"hasCooling": true,
"hasAuto": false,
"hasBoost": false,
"c1Demand": false,
"c2Demand": false,
"doingBoost": false,
"timerRunning": false,
"fixAvailable": true
```

Reply:
Ok so no idea if we can get somewhere the available feature (cooling, heating, ...) to properly configure the integration.
I'm going to accept in this way right now...